addon/components/nrg-application/template.hbs

```diff
@@ -1,12 +1,12 @@
 {{! template-lint-disable no-inline-styles }}
 {{nrg-appbar title=title toggleSidebar="toggleSidebar"}}
 <div class="pushable">
-  {{nrg-sidebar isOpen=sidebarIsOpen context=(concat "#" elementId " .pushable")}}
+  {{nrg-sidebar isOpen=sidebarIsOpen context=(concat "#" elementId " .pushable") clicked=(action clicked)}}
   {{#if fullscreenMap}}
     <div class="pusher">
       {{#if sidebarMenuIsOpen}}
         <div class="sidebar-solid">
-          {{nrg-sidebar-menu}}
+          {{nrg-sidebar-menu clicked=(action clicked)}}
```

Comment:
If someone didn't implement the `clicked` function, there would be a template error.
```hbs
{{nrg-sidebar-menu clicked=(action "clickedLink")}}
```

```js
clickedLink((item) {
  if (this.clicked) {
    this.clicked(item);
  }
}
```